pkg/api/tests.go

```diff
@@ -239,9 +239,16 @@ func BuildTestsResults(dbc *db.DB, release, period string, fil *filter.Filter) (
 				current_successes * 100.0 / NULLIF(current_runs, 0) AS current_pass_percentage,
 				current_failures * 100.0 / NULLIF(current_runs, 0) AS current_failure_percentage,
 				current_flakes * 100.0 / NULLIF(current_runs, 0) AS current_flake_percentage,
+				(current_successes + current_flakes) * 100.0 / NULLIF(current_runs, 0) AS current_working_percentage,
+
 				previous_successes * 100.0 / NULLIF(previous_runs, 0) AS previous_pass_percentage,
 				previous_failures * 100.0 / NULLIF(previous_runs, 0) AS previous_failure_percentage,
 				previous_flakes * 100.0 / NULLIF(previous_runs, 0) AS previous_flake_percentage,
+				(previous_successes + previous_flakes) * 100.0 / NULLIF(previous_runs, 0) AS previous_working_percentage,
+
+				(current_failures * 100.0 / NULLIF(current_runs, 0)) - (previous_failures * 100.0 / NULLIF(previous_runs, 0)) AS net_failure_improvement,
+				(current_flakes * 100.0 / NULLIF(current_runs, 0)) - (previous_flakes * 100.0 / NULLIF(previous_runs, 0)) AS net_flake_improvement,
```

Comment:
Using _improvement for these two above looks wrong to me, if current fail or flake percentage is 70%, and it used to be 50% previously, we're reporting 20% growth as an "improvement" when things are actually getting worse. Maybe just need to make it previous - current instead of current - previous?

Reply:
Makes sense, I flipped them